README.md

```diff
@@ -103,26 +103,56 @@ Unless otherwise specified, all inputs and outputs will be JSON serialised.
   - Output: A single-use UUID4.
 - `POST /awala`: [Awala endpoint middleware](https://github.com/relaycorp/relayverse/issues/28) backend.
   - Auth: Awala Endpoint Middleware.
-  - Incoming service messages:
-    - `MemberIdRequest`. Payload (`application/json`):
-      - The id for the respective member public key.
-      - The current timestamp.
-      - Digital signature for the parameters above, produced with the private key associated with the public key.
-    - `MemberPublicKeyImport`. Payload (`application/json`):
-      - The single-use import token.
-      - The DER-encoded public key.
-  - Outgoing service messages:
-    - `MemberIdBundle`.
-      - Trigger: Successful `MemberIdRequest`.
-      - Payload (`application/vnd.veraid.member-bundle`): VeraId Member Bundle.
-    - `MemberPublicKeyImportAck`.
-      - Trigger: Successful `MemberPublicKeyImport`.
-      - Payload (`application/json`):
-        - The id for the member public key. This is to be passed in subsequent `MemberIdRequest` messages.
-        - VeraId Member Bundle.
+  - HTTP response: `202 Accepted` (no content) if the input was valid and the request was successfully processed, or `400 Bad Request` if the input was invalid.
+  - Awala service messages:
+    - `MemberBundleRequest`.
+      - HTTP request body (JSON, with content type `application/vnd.veraid.member-bundle-request`):
+        - The id for the respective member public key.
+        - The future start date of the bundle. (Bundles will be issued at that time or later, but never before)
+        - Digital signature for the parameters above, produced with the private key associated with the public key.
+        - The Awala Parcel Delivery Authorisation (PDA).
+      - Successful outcome: Create new DB record to **schedule** the issuance and delivery of a member id bundle. This DB record will contain the data in the request.
+        - Any public key must have a maximum of 1 request at any time, so if we get a duplicate, we should replace the old request with the new one.
```

Comment:
We are deleting it from the DB right?

Reply:
Not deleting per se -- upserting. Example: https://github.com/relaycorp/relaynet-internet-gateway/blob/1e1ea540cf989f2ff4a5cb3d71ad8c14f9236094/src/utilities/config.ts#L19